tools/lab_dispatch/generic_source.yaml

```diff
@@ -0,0 +1,78 @@
+job_queue: $INPUT_QUEUE
+global_timeout: 3600
+output_timeout: 1800
+provision_data:
+  $INPUT_DATA_SOURCE
+test_data:
+  attachments:
+    - local: "tools/lab_dispatch/resources/manifest.conf"
+      agent: "manifest.conf"
+    - local: "tools/lab_dispatch/resources/input.conf"
+      agent: "input.conf"
+    - local: "tools/lab_dispatch/resources/service_file.service"
+      agent: "service_file.service"
+    - local: "tools/lab_dispatch/build_install_deb.py"
```

Comment:
Not to action right now, but if there's any chance this `build_install_deb` could be useful in non-Checkbox workflows, then it might belong to the tools repo and eventually move there.

Reply:
I don't see a plausible usage of this outside of checkbox as it is very checkbox-oriented right now, and will become more so in the future. I would leave this here. If we find we need it somewhere else, we can move it.